src/app/Analytics/LocalMonitoring/RecommendationsForLocalMonitoring/RemoteMonitoring/RecommendationComponents/CostDetails.tsx

```diff
@@ -87,9 +88,47 @@ const CostDetails = (props: { recommendedData; currentData; chartData; day; endt
       NumberFormat(props.recommendedData[0]?.recommendation_engines?.cost?.variation?.limits?.cpu?.amount)
     )}`;
 
+  /// Alert for over and under utlized
+  useEffect(() => {
+    if (props.recommendedData !== null) {
+      utilizationAlert(props.recommendedData);
+    }
+  }, [props.recommendedData]);
+
+  type alertVarient =  'success' | 'danger' | 'warning' | 'info' | 'custom';
+ 
+  const [showSuccessAlert, setShowSuccessAlert] = useState(false);
+  const [alertMessage, setAlertMessage] = useState('');
+  const [notificationType, setNotificationType] = useState<alertVarient>('info');
```

Comment:
Is it alertVariant ?

Reply:
this ` useState<alertVarient>` is the same `type alertVarient` on line 98 hence the same spelling. (I guess the spelling i can correct to 'variant' at both places)